tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -20,6 +23,37 @@ def __init__(self):
         super(GlusterIntegrationSdsSyncStateThread, self).__init__()
         self._complete = gevent.event.Event()
 
+    def _emit_event(self, resource, curr_value, msg):
+        alert = {}
+        alert['source'] = 'tendrl-gluster-integration'
+        alert['pid'] = os.getpid()
+        alert['timestamp'] = tendrl_now().isoformat()
+        alert['alert_type'] = 'status'
+        severity = "info"
+        if curr_value == "Stopped":
+            severity = "critical"
+        alert['severity'] = severity
+        alert['resource'] = resource
```

Comment:
@r0h4n The alerting module expects it as volume_status or volume_utilization so basically its the combination of <entity_type>_<alert_type> where alert_type is either status or utilization